source/simulator/assemblers/newton_stokes.cc

```diff
@@ -367,12 +367,12 @@ namespace aspect
                                                 eta * 2.0 * (scratch.grads_phi_u[i] * scratch.grads_phi_u[j])
                                                 // assemble \nabla p as -(p, div v):
                                                 - (pressure_scaling *
-                                                   scratch.div_phi_u[i] * scratch.phi_p[j])
+                                                   (scratch.div_phi_u[i] * scratch.phi_p[j]))
```

Comment:
It should make no difference -- both quantities are scalars, so at least from a mathematical perspective, this will have no effect (but is also not wrong).

Reply:
ah, yes, you are right. I will remove it.

Reply:
I don't think there's harm in keeping it. Do as you think is easiest to read.